common/test/java/com/couchbase/lite/BaseDbTest.java

```diff
@@ -1655,14 +1665,38 @@ protected final void verifyDocument(DictionaryInterface doc) {
         verifyBlob(doc.getBlob("doc-29"));
     }
 
+    protected Database openDatabase() throws CouchbaseLiteException { return verifyDb(createDb("test_db")); }
```

Comment:
Please use the name randomizer, here.  We've had a a lot of trouble, in the past, with tests failing because of some cruft left around by previous tests.